pkg/services/rpcsrv/server_test.go

```diff
@@ -1692,12 +1838,19 @@ var rpcTestCases = map[string][]rpcTestCase{
 				}
 			},
 		},
+		{
+			name:    "no params",
+			params:  "[]",
+			fail:    true,
+			errCode: neorpc.ErrInvalidParams.Code,
+		},
 	},
 }
 
 func TestRPC(t *testing.T) {
 	t.Run("http", func(t *testing.T) {
 		testRPCProtocol(t, doRPCCallOverHTTP)
+		testFunctionsWithOldStates(t, doRPCCallOverHTTP)
 	})
```

Comment:
Actually, it works well on my machine and can be debugged properly. Currently it fails, but it can be fixed:
```
=== RUN   TestRPC/http/test_func_with_old_states/getstate
            --- FAIL: TestRPC/http/test_func_with_old_states/getstate (0.00s)
=== RUN   TestRPC/http/test_func_with_old_states/getstate/no_params
=== CONT  TestRPC/http/test_func_with_old_states/getstate/no_params
    server_test.go:2787: 
        	Error Trace:	/home/anna/Documents/GitProjects/nspcc-dev/neo-go/pkg/services/rpcsrv/server_test.go:2787
        	            				/home/anna/Documents/GitProjects/nspcc-dev/neo-go/pkg/services/rpcsrv/server_test.go:3533
        	Error:      	Not equal: 
        	            	expected: -701
        	            	actual  : -32602
        	Test:       	TestRPC/http/test_func_with_old_states/getstate/no_params
                --- FAIL: TestRPC/http/test_func_with_old_states/getstate/no_params (0.00s)


Expected :-701
Actual   :-32602
```

The only advice I can give you is try to run this test in a separate suite:
```

func TestRPCOldStates(t *testing.T) {
	t.Run("http", func(t *testing.T) {
		testFunctionsWithOldStates(t, doRPCCallOverHTTP)
	})
}
```

Reply:
Thanks, I don't know what happened, reboot helped)